doc/source/whatsnew/v1.3.0.rst

```diff
@@ -720,7 +720,7 @@ Other
 - Bug in :meth:`DataFrame.equals`, :meth:`Series.equals`, :meth:`Index.equals` with object-dtype containing ``np.datetime64("NaT")`` or ``np.timedelta64("NaT")`` (:issue:`39650`)
 - Bug in :func:`pandas.util.show_versions` where console JSON output was not proper JSON (:issue:`39701`)
 - Bug in :meth:`DataFrame.convert_dtypes` incorrectly raised ValueError when called on an empty DataFrame (:issue:`40393`)
-
+- Bug in aggregation functions for :class:`DataFrame` not respecting ``numeric_only`` argument when ``level`` keyword was given (:issue:`40660`)
```

Comment:
level is really doing a groupby, I think it make senses to move the note there.

Reply:
yeah, generally try to avoid the 'Other' section.

nb. we have a PR somewhere deprecating `level=` to these

Reply:
Thx, was not sure if groupby was the correct place. 

This bug is actually a good argument for the deprecation and removal...